proto/cosmos/nft/v1beta1/genesis.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package cosmos.nft.v1beta1;
+
+import "cosmos/nft/v1beta1/nft.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/nft";
```

Comment:
how about putting genesis into a subpackage?

Reply:
Do you mean that genesis output to `github.com/cosmos/cosmos-sdk/x/nft/genesis`?

Reply:
yes. What do you think about this idea?

Reply:
When thinking more about it - I would propose to split it in the following way: 
* x/nft -- for all things related to RPC
* x/ecocredit/dal -- for all things related to storage (data access layer)
This way we have a clear domains.

Reply:
BTW: Aaron pointed in a chat, that this will require splitting the proto packages as well:
*  proto/cosmos/nft/v1beta1
*  proto/cosmos/nft/dal/v1beta1

Reply:
I think this is a good idea. If so, is it necessary to replace the genesis package with `x/nft/conf` to store genesis and param as the module configuration package. The adjusted structure is as follows:

- `x/nft`   -- for all things related to RPC
  - tx.pb.go
  - query.pb.go
  - ....
- `x/nft/dal ` --  for all things related to storage 
  - keeper.go
  - grpc_query.go
  - store_key.go
  - ......
- `x/nft/conf` -- for all things related to config
  - genesis.pb.go
  - params.pb.go
- `x/nft/client`
- `x/nft/simulation`

Reply:
The main point of breaking down the proto into 2 (or more) packages is to have more freedom to upgrading it in the future:
* the protobuf service API and related types should be rather stable and we want to avoid any breaking change. We will version it.
* the storage types can be in an internal package and don't need to be versioned. If "tomorrow" we will find a better storage layout we can update the module without bumping the API version.

Reply:
Okay, got it

Reply:
Let's please not use the term dal. I would use state instead. I also don't want to delay this PR unnecessarily, and would prefer to do this design in a more general way for the SDK.

Reply:
If so, how can this PR be modified?

Reply:
Recently, @aaronc  pointed that the store proto version is important for merkle proofs. So, we will probably need to discuss more about it. 
TL;DR - let's merge this in one package.